manifests/a/AnyLogic/AnyLogic/PLE/8.7.12/AnyLogic.AnyLogic.PLE.installer.yaml

```diff
@@ -0,0 +1,22 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS2.7-2-5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: AnyLogic.AnyLogic.PLE
+PackageVersion: 8.7.12
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0
+InstallerType: nullsoft
+UpgradeBehavior: install
+FileExtensions:
+- alp
+- java
+ReleaseDate: 2022-05-25
+Installers:
+- Architecture: x64
+  InstallerUrl: https://files.anylogic.com/anylogic-ple-8.7.12.x86_64.exe
+  InstallerSha256: E13F6AAB83A191B5A892325FEA112E2F4FD9618BA176ECF339F3E9D2C2DCF2CF
+- Architecture: x86
+  InstallerUrl: https://files.anylogic.com/anylogic-ple-8.7.12.x86_64.exe
+  InstallerSha256: E13F6AAB83A191B5A892325FEA112E2F4FD9618BA176ECF339F3E9D2C2DCF2CF
```

Comment:
```suggestion
```
![image](https://user-images.githubusercontent.com/12611259/178646326-dbfe21f7-ab63-4248-bd06-dde2028b202b.png)

Reply:
Oh, I assumed the installer works for x86 as the name of the installer suggested so. Would check this next time in a 32-bit VM.